app/controllers/suggestions.rb

```diff
@@ -0,0 +1,75 @@
+get '/suggestions' do
+  @suggestions = Suggestion.all
+  @suggestions_by_votecount = @suggestions.sort_by{|suggestion| suggestion.votes.count}.reverse
```

Comment:
There is definitely a way to do this with SQL, which would be much faster as your database grows. Challenge: write out this query in straight SQL, then utilize ActiveRecord methods `join` `group_by` `order` to write the query.